src/assets/i18n/fr.json5

```diff
@@ -3018,51 +3486,50 @@
   // "menu.section.icon.control_panel": "Control Panel menu section",
   "menu.section.icon.control_panel": "Section du menu relative au panneau de configuration",
 
-  // "menu.section.icon.curation_task": "Curation Task menu section",
-  "menu.section.icon.curation_task": "Section du menu relative aux tâches de Curation",
+  // "menu.section.icon.curation_tasks": "Curation Task menu section",
+  "menu.section.icon.curation_tasks": "Section du menu relative aux tâches de curation",
 
   // "menu.section.icon.edit": "Edit menu section",
   "menu.section.icon.edit": "Section du menu relative à l'édition",
 
   // "menu.section.icon.export": "Export menu section",
-  "menu.section.icon.export": "Section du menu relative à l'export",
+  "menu.section.icon.export": "Section du menu relative à l'exportation",
 
   // "menu.section.icon.find": "Find menu section",
   "menu.section.icon.find": "Section du menu relative à la recherche",
 
   // "menu.section.icon.import": "Import menu section",
-  "menu.section.icon.import": "Section du menu relative à l'import",
+  "menu.section.icon.import": "Section du menu relative à l'importation",
 
   // "menu.section.icon.new": "New menu section",
   "menu.section.icon.new": "Section du menu relative à l'ajout d'éléments",
 
   // "menu.section.icon.pin": "Pin sidebar",
-  "menu.section.icon.pin": "Epingler menu latéral",
+  "menu.section.icon.pin": "Épingler le menu latéral",
 
   // "menu.section.icon.processes": "Processes menu section",
-  "menu.section.icon.processes": "Section du menu relatif aux Processus",
+  "menu.section.icon.processes": "Section du menu relatif aux processus",
 
   // "menu.section.icon.registries": "Registries menu section",
-  "menu.section.icon.registries": "Section du menu relative aux Registres",
+  "menu.section.icon.registries": "Section du menu relative aux registres",
 
   // "menu.section.icon.statistics_task": "Statistics Task menu section",
-  "menu.section.icon.statistics_task": "Section du menu relative aux tâches Statistiques",
-
-  // "menu.section.icon.unpin": "Unpin sidebar",
-  "menu.section.icon.unpin": "Libérer menu latéral",
+  "menu.section.icon.statistics_task": "Section du menu relative aux tâches statistiques",
 
+  // "menu.section.icon.workflow": "Administer workflow menu section",
+  "menu.section.icon.workflow": "Section du menu relative à l'administration du Workflow",
```

Comment:
Linked to my previous comment on "Administer workflow" translation. Even if long, I think : "Section du menu relative à la gestion des dépôts en traitement." may be better.

Reply:
@pilasou Would you agree with  "Section du menu relative à la gestion des soumissions en Workflow"?

Reply:
@mhvezina Yes, your wording is better. I agree with "Section du menu relative à la gestion des soumissions en Workflow"